.github/CONTRIBUTING.md

```diff
@@ -42,9 +42,9 @@ For new feature requests, please elaborate on the context and the benefit the fe
 
 ## Pull requests
 
-### Github Flow
+### GitHub Flow
 
-This repository uses the [Github Flow](https://docs.github.com/en/get-started/quickstart/github-flow) model for collaboration. To submit a pull request:
+This repository uses the [GitHub Flow](https://docs.github.com/en/get-started/quickstart/github-flow) model for collaboration. To submit a pull request:
```

Comment:
This particular file has been copied from `r.pkg.template` repo into all of other repos. If you want to propose changes there I would open a PR and then ask IDR to propagate.

Reply:
Thanks. Didn't know about it. Opened a PR here https://github.com/insightsengineering/r.pkg.template/pull/157